cherryontech/birthdaybasher
# feat: Add styling and date

This PR starts styling the index and card pages.

**Work done**
- Starts styling the card page
- Add date to card page
- Remove boilerplate from index
- Add CoT's custom logo as favicon

**Steps to test**
1. Serve gridsome locally
2. Navigate to the index page (usually http://localhost:8080/)
- See the new custom CoT content
3. Navigate to a card page, such as [this one](http://localhost:8080/card/randompath1).
- See the date at the top of the page
- See layout with padding at breakpoints, as per figma.

**Things I learned/Gotchas**
This really should have been multiple PRs, but I got carried away with the styling! 😊
